open-company/open-company-web
# App update banner refresh

Card: https://trello.com/c/z62uhpLg

To test:
- change line 10 and 11 of `oc.web.actions.app-update` and set 3 and 60 seconds for example
- open the app
- can you see the app update happening every 3 seconds? Good
- now delete this file `rm resources/public/version/version.json` and wait for rebuild
- open the web app
- do you see the banner to refresh the app? Good
- dismiss it with the X
- do you NOT see the app update refresh every 3 seconds anymore? (it should be delayed by 60 seconds now)
- [ ] after 60 seconds do you see the banner again? Good
- [ ] try the refresh button
